check_release_assets.py

```diff
@@ -22,35 +22,45 @@
 
 sdist_project = 'cupy'
 
+_v13_cuda_matrix = list(itertools.product(
+    (CP38, CP39, CP310, CP311), (LINUX, WINDOWS)))
+_v13_aarch64_matrix = list(itertools.product(
+    (CP38, CP39, CP310, CP311), (LINUX_AARCH64,)))
+_v13_rocm_matrix = list(itertools.product(
+    (CP38, CP39, CP310, CP311), (LINUX,)))
```

Comment:
Python 3.8 support will be dropped in CuPy v13.
```suggestion
_v13_cuda_matrix = list(itertools.product(
    (CP39, CP310, CP311), (LINUX, WINDOWS)))
_v13_aarch64_matrix = list(itertools.product(
    (CP39, CP310, CP311), (LINUX_AARCH64,)))
_v13_rocm_matrix = list(itertools.product(
    (CP39, CP310, CP311), (LINUX,)))
```